backend/src/main/java/com/example/cms/configuration/ApplicationConfigurationProvider.java

```diff
@@ -41,6 +42,7 @@ public ApplicationConfigurationProvider(
             @Value("${app.typesense.host}") String typesenseHost,
             @Value("${app.path.uploads}") String uploadsDirectory,
             @Value("${app.path.backups}") String backupsDirectory,
+            @Value("${EMAIL_TEMPLATE:/emailTemplates/}") String emailTemplatesDirectory,
```

Comment:
tu jeszcze możesz zrobić żeby to było w pliku META-INF opisane jako app.path.templates i tak to potem w application.yml wsadzić i będzie gitara